core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -598,6 +598,88 @@ class DAGSchedulerSuite
     assertDataStructuresEmpty()
   }
 
+  test("run with ShuffleMapStage retry") {
```

Comment:
ha, now I see that andrew asked for a more concise test name earlier, though I'm not sure if we can come up w/ a 5 word decription that is precise enough ...